packages/esm-form-entry-app/src/app/form-schema/form-schema.service.ts

```diff
@@ -27,11 +40,26 @@ export class FormSchemaService {
     }).pipe(
       map(({ unCompiledSchema, formMetadataObject }) => {
         const formSchema: any = unCompiledSchema.form;
+
+        if (formSchema?.translations) {
+          formSchema.translations.forEach((translationUuid: string) => {
+            this.getFormTranslationResourceByUuidFromServer(translationUuid)
+              .then((result) => {
+                this.cacheTranslationsByUuid(translationUuid, result);
+                this.translate.setTranslation(result.language, result.translations, true);
+              })
+              .catch((error) => {
+                console.error(`Error getting AMPATH translation resource ${JSON.stringify(error)}.`);
```

Comment:
It's good to have a `catch()` here. It would be better if we logged the actual error rather than calling `JSON.stringify()` on it. Also, should this display a message to the user?